app/konnect/declarative-config.md

```diff
@@ -0,0 +1,206 @@
+                paths:
+                - /mockpath
+    ```
+
+    Every service must have at least one initial version.
+
+3. Check that the format is correct and that decK recognizes that you've made
+changes:
+
+    ```sh
+    deck konnect diff --konnect-email <email> --konnect-password-file <pass>.txt
+    ```
+
+    If the format and schema is correct, decK gives you a preview of what will
+    be added to the control plane configuration:
+
+    ```sh
+    creating service 9595B5F9-3B6A-4C48-BE93-9EC1B0EA487A
+    creating route mockpath
+    creating service-package MyService
+    creating service-version 1
+    Summary:
+      Created: 4
+      Updated: 0
+      Deleted: 0
+    ```
+
+4. If you're satisfied with the preview, sync the changes to the control plane:
+
+    ```sh
+    deck konnect sync --konnect-email <email> --konnect-password-file <pass>.txt
+    ```
+
+    You should see the same output again:
+
+    ```sh
+    creating service 9595B5F9-3B6A-4C48-BE93-9EC1B0EA487A
+    creating route mockpath
+    creating service-package MyService
+    creating service-version 1
+    Summary:
+      Created: 4
+      Updated: 0
+      Deleted: 0
+    ```
+
+5. Check Konnect Cloud to make sure the sync worked. You should see a
+new service named `MyService` in the ServiceHub.
+
+    ![ServiceHub tiles](/assets/images/docs/konnect/konnect-myservice.png)
+
+
+## Add global configuration
+
+While you can't fully edit global configuration through the Konnect Cloud GUI yet,
+you can do it with decK.
+
+1. Add a consumer to the `konnect.yaml` file:
+
+    ```yaml
+    consumers:
+     - custom_id: consumer
+       username: consumer
+       keyauth_credentials:
+       - key: apikey
+    ```
+
+2. Add a global proxy cache plugin:
+
+    ```yaml
+    plugins:
+    - name: proxy-cache
+      config:
+        content_type:
+        - "application/json; charset=utf-8"
+        cache_ttl: 30
+        strategy: memory
+    ```
+
+3. Run a diff to test your changes:
+
+    ```sh
+    deck konnect diff --konnect-email <email> --konnect-password-file <pass>.txt
+    ```
+
+4. If everything looks good, run another sync, then check Konnect to see your
+changes:
+
+    ```sh
+    deck konnect sync --konnect-email <email> --konnect-password-file <pass>.txt
+    ```
+
+    You can find the global configuration under the
+    ![icon](/assets/images/icons/konnect/konnect-shared-config.svg){:.inline .no-image-expand}
+    **Shared Config** menu option.
+
+    ![Shared config overview](/assets/images/docs/konnect/konnect-shared-config.png)
+
+    {:.note}
+    > **Note:** If you add consumers to the `konnect.yaml` file and sync your
+    local file to Konnect, it picks up consumers automatically. However, if you
+    want to pull consumer configuration down **from** Konnect, use the
+    `--include-consumers` flag in your request.
+
+## Next steps
+
+If you haven't already, [set up a runtime](/konnect/runtime-manager/) to start
+proxying your services.
```

Comment:
Are these the right next steps? In the TOC in the Nav bar the next items is using the Konnect API.